src/answers-umd.js

```diff
@@ -231,7 +216,26 @@ class Answers {
         urlWithoutQueryParamsAndHash(document.referrer)
       );
     }
+    return storage;
+  }
+
+  /**
+   * Initializes the SDK with the provided configuration. Note that before onReady
+   * is ever called, a check to the relevant Answers Status page is made.
+   *
+   * @param {Object} config The Answers configuration.
+   * @param {Object} statusPage An override for the baseUrl and endpoint of the
+   *                            experience's Answers Status page.
+   */
+  init (config, statusPage) {
```

Comment:
can we have a pr after this to remove the statusPage param?

Reply:
sure